quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -25,10 +25,10 @@ use std::time::Duration;
 use anyhow::Context;
 use async_trait::async_trait;
 use futures::stream::FuturesUnordered;
-use futures::StreamExt;
+use futures::{Future, StreamExt};
 use quickwit_actors::{
-    Actor, ActorContext, ActorExitStatus, ActorHandle, Handler, Mailbox, Supervisor, Universe,
-    WeakMailbox,
+    Actor, ActorContext, ActorExitStatus, ActorHandle, DeferableReplyHandler, Handler, Mailbox,
```

Comment:
`DeferableReplyHandler` is also something introduced to support ingest v1 that we should not forget to get rid of eventually.

Reply:
I think it is actually quite nice.